### PR TITLE
Updated my octocat with a caption.

### PR DESCRIPTION
**Pull request by lost-grail**
Pretty mediocre by the rest of your standards

### CODE REVIEW

**Comment by bkentel-mcafee** [comment]
You should reference the relevant issue in your commit message.

File: _posts/0000-01-12-lost-grail.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (khiepleh-mfe) [inline _posts/0000-01-12-lost-grail.md]: You _also_ forgot to update the title 😱

**Comment by lost-grail** [comment]
What was the accepted _minimum_ again?

**Comment by bkentel-mcafee** [comment]
@khiepleh-mfe the required number was 5, correct?

**Comment by khiepleh-mfe** [comment]
> 
> 
> What was the accepted _minimum_ again?

5, but we here at _McAfee_ don't cut corners, right? There's also not _technically_ a quota for unique  emojis, but, well... you don't want to be one of _those_ people.

**Comment by lost-grail** [comment]
☣️ closes #10

**Pull request merged**